src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I apologize, what I meant was that prior to acceptance I wanted to try similar patches (which include but are not limited to these lines of code).

First I wanted to attempt the simplification you indicated, although such a simplification can always be added later.

I agree that the basic idea of a patch should be understandable - and my reading of the idea for this patch is, "If the opponent is forced to make a capture in response to this move, then reduce LMR reduction by 1 instead of what it would have otherwise been reduced by, just like the `captureOrPromotion` branch."  If you look upward a few lines, you see the condition `&& (!captureOrPromotion || moveCountPruning)` indicating that the `captureOrPromotion` branch would only get executed during move-count pruning.  It's true in general that it's difficult to understand why one patch outperforms another; here especially so as I'm still unclear on the basic idea (my interpretation seems not understandable, but I don't see a simpler interpretation)

It's important that I improve w/ a similar patch (or at least see if such is possible) as then we may be able to explain reasons the patch gains Elo.  As I'm currently looking at this, we're changing one of the most complex aspects of search which we had previously indirectly changed (w/respect to `see_ge` below), and we found this change by trial-and-error but then made no attempt to explain how this patch works.

Pros:
* The motivation for the patch (captures have some effect on LMR) is plausible
* The patch gains Elo

Cons:
* Due to the immediately-enclosing context being complex, this patch isn't simple (although the number of lines changed and the characters changed are simple)
* We already know that what works well for standard chess often doesn't work well for giveaway chess, and yet this patch is a copy-and-paste-and-replace of existing code for standard chess... which I guess is OK, after all we need to start somewhere, but we should have expected `anti_lmr5` to work as well as the previous 4 changes.  Sure, the Elo-gaining test results are reliable, but now I'm terrified to accept further giveaway patches as this change almost as destabilizing as `anti_lmr6`.